src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkImageBrowser.tsx

```diff
@@ -138,7 +138,7 @@ export class SmallArtworkImageBrowser extends React.Component<
                   enabled={image.is_zoomable}
                   isDefault={image.is_default}
                   src={image.uri}
-                  maxHeight={image.height}
+                  maxHeight="2000px"
```

Comment:
Why 2000 here, versus 3000 up above?

Reply:
This ArtworkImageBrowser is for mobile, so extra large screens aren't really a concern.

Reply:
ArtworkImageBrowser is also for desktop

Reply:
There are 2 in the component, `SmallArtworkImageBrowser` and `SmallArtworkImageBrowser`. But I've decided to make it 2000px for both.